src/services/stag/ExamService.js

```diff
@@ -0,0 +1,98 @@
+const BaseService = require('./BaseService');
+
+class ExamService extends BaseService{
+
+    /**
+     * @param {StagAdapter} stagAdapter
+     */
+    constructor(stagAdapter) {
+        super(stagAdapter);
+        this.serviceEndpoint = 'terminy';
+    }
+
+    /**
+     * Vrací zkouškové termíny pro přihlášeného studenta
+     * Z mandatory bloku je třeba vybrat alespoň jeden prvek pro filtrovani
+     *
+     * @param {Object}  [mandatory]
+     * @param {string}     [mandatory.zkratka]
+     * @param {string}     [mandatory.katedra]
+     * @param {string}     [mandatory.rok]
+     * @param {string}     [mandatory.osCislo]
+     * @param {Object}  [optional]
+     * @param {boolean}    [optional.zobrazitProsle]
+     * @param {boolean}    [optional.zobrazitZrusene]
+     * @param {boolean}    [optional.zobrazitBlokovane]
+     *
+     * @return {Promise<ExamEvents>}
+     */
+    getExamEvents(mandatory = {}, optional = {}){
```

Comment:
Mandatory bych pojmenoval jinak, něco jako **filterBy**. To že je ten parametr nutný by mělo být promítnutý tím, že není v docblocku v hranatých závorkách.